packages/pluggableWidgets/datagrid-web/src/Datagrid.tsx

```diff
@@ -95,6 +99,7 @@ export default function Datagrid(props: DatagridContainerProps): ReactElement {
             paging={props.pagingEnabled}
             pagingPosition={props.pagingPosition}
             settings={props.configurationAttribute}
+            onSettingsChange={onConfigurationChange}
```

Comment:
In this way we are always creating a closure, even having or not the property set. Maybe only create if the props is set?

Reply:
Fixed